rules.d/60-cryptsetup-id.rules

```diff
@@ -0,0 +1,10 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="cryptsetup_id_end"
+SUBSYSTEM!="block", GOTO="cryptsetup_id_end"
+
+KERNEL=="dm-*", IMPORT{program}="cryptsetup_id"
```

Comment:
hmm, the canonical way to detect the type of DM device is via checking for `DM_UUID=` starting with `CRYPT-` (and similar). i don't think we should bother with other DM devices here, i.e. some LVM stuff or so.